samples/buildstrategy/buildstrategy_buildpacks-v3_cr.yaml

```diff
@@ -71,4 +72,18 @@ spec:
       volumeMounts:
         - name: layers-dir
           mountPath: /layers
-    
\ No newline at end of file
+    - name: step-export
+      image: $(build.builderImage)
+      securityContext:
+        runAsUser: 0
```

Comment:
We could optimize this later.

Reply:
The reason for the `runAsUser: 0` is the following error: ```